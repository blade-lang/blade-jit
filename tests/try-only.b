try {
  raise new Error('Something happened')
}