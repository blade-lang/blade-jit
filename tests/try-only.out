^^^
try must declare at least one of `catch` or `finally`